openapi/src/parts/controlled_aws_sagemaker_notebook.yaml

```diff
@@ -0,0 +1,232 @@
+paths:
+  /api/workspaces/v1/{workspaceId}/resources/controlled/aws/sagemaker-notebooks:
```

Comment:
I would suggest not embedding "sagemaker" in the endpoint path. It is awkward and it is a marketing name. As we have seen with "AI Notebooks", not necessarily stable over time.